netplay/createroom.cpp

```diff
@@ -39,20 +40,32 @@ CreateRoom::CreateRoom(QWidget *parent)
         playerNameEdit->setText(w->getSettings()->value("netplay_name").toString());
     layout->addWidget(playerNameEdit, 3, 1);
 
+    QLabel *useInputDelayLabel = new QLabel("Use Fixed Input Delay?", this);
+    layout->addWidget(useInputDelayLabel, 4, 0);
+    useInputDelay = new QCheckBox(this);
+    layout->addWidget(useInputDelay, 4, 1);
+    connect(useInputDelay, SIGNAL (clicked(bool)), this, SLOT(handleUseInputDelay(bool)));
+
+    QLabel *inputDelayLabel = new QLabel("Input Delay", this);
+    layout->addWidget(inputDelayLabel, 5, 0);
+    inputDelay = new QLineEdit(this);
```

Comment:
It wouldn't hurt to add:

```inputDelay ->setValidator( new QIntValidator(0, 100, this) );```

Here. This is an input validator, which would only allow integer values between 0 and 100 (I don't know what the max should be). This would just prevent people from causing weird issues by entering text or negative numbers